pkg/apis/extensions/types.go

```diff
@@ -226,6 +226,14 @@ type DeploymentSpec struct {
 	// Value of this key is hash of DeploymentSpec.PodTemplateSpec.
 	// No label is added if this is set to empty string.
 	UniqueLabelKey string `json:"uniqueLabelKey,omitempty"`
+
+	// The config this deployment is rolling back to. Will be cleared after rollback is done.
+	Rollback *RollbackConfig `json:"rollback,omitempty"`
```

Comment:
How about adding a new field `RollbackVersion`
```go
type DeploymentSpec struct {
	...
	// The version this deployment is rolling back to. Will be cleared after rollback is done.
	RollbackVersion int
}
```
And we pass `RollbackConfig` and copy `RollbackConfig.Version` to `DeploymentSpec.RollbackVersion` when rolling back, like how `Scale` works?